pkg/client/introduce/client.go

```diff
@@ -27,8 +33,10 @@ type Provider interface {
 // Client enable access to introduce API
 type Client struct {
 	service.Event
-	service service.DIDComm
-	store   storage.Store
+	service    service.DIDComm
+	store      storage.Store
+	defaultInv *didexchange.Invitation
```

Comment:
Where is this set ? I see only one [reference](https://github.com/hyperledger/aries-framework-go/pull/737/files#diff-53f48d9400b55325b3f6a88c6129625eR86) where we are assigning this to opts.

Reply:
Correct, there is no possibility to set it. It will be added on the next PR.

Reply:
@rolsonquadras I've added a function `SetDefaultInvitation`

Reply:
I don't think we should mutate default invitation after creation of the client.

Reply:
@troyronda totally agree I was thinking about it. My decision was based on flexibility.

Reply:
Done!